app/src/main/res/layout/fragment_gamescreen_five_players.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".fragments.GameScreenFivePlayersFragment">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <ImageView
+            android:id="@+id/gamescreenTriangleImageView"
+            android:layout_width="150dp"
+            android:layout_height="115dp"
+            android:rotation="-180"
+            android:scaleType="fitCenter"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:srcCompat="@drawable/star" />
+
+        <androidx.fragment.app.FragmentContainerView
+            android:id="@+id/player1FragmentContainerView"
+            android:name="at.aau.serg.fragments.CardsFragment"
```

Comment:
I don't quite understand why we use the `CardsFragment`, which displays a list of cards if we only want to display a single card.

Reply:
have now changed it to ImageViews